src/TextEdit/TextEditMode.js

```diff
@@ -1,10 +0,0 @@
-/**
```

Comment:
Note: when moving a file in a git repo it is better to use `git mv` so that git registers the file as being renamed and keeps tracking changes. Otherwise it is likely to just mark the previous name as being deleted and the new name as being a completely new file rather than a pre-existing file with a new name.